### ISSUE TITLE
[Bug]: Wrong background color in editor windows

### ISSUE THREAD

**Issue by apb-hello-world** [mallowigi/material-theme-issues#484]
### Describe the bug

I have chosen the material deep ocean theme and everything looks as it does in the preview except the editor/console background colors. This happens in all Jetbrains IDEs. It is not happening on my personal machine, just on my work computer and I pulled the settings down from the server.

![hXdoLnLYVz](https://github.com/mallowigi/material-theme-issues/assets/8071683/42d34cd4-4c63-4929-9db8-f35f4b33c5ee)

### To Reproduce

1. Set theme to Material Deep Ocean

### Expected behavior

Theme should match preview. 

### Material Theme Version

8.13.2

### IDE

WebStorm, DataGrip

**Comment by mallowigi** [comment]
Are you using Settings Sync?

**Comment by apb-hello-world** [comment]
> Are you using Settings Sync?

Yeah. I pushed my settings up from my home machine, and pulled them down on my work machine. I just opened up webstorm on my home machine and it is now having the same problem. This is what it looked like last night when I first set up material UI:

![image](https://github.com/mallowigi/material-theme-issues/assets/8071683/4a57c85c-9824-4583-ad57-faf4b9bb94ba)

Now it looks like this:

![image](https://github.com/mallowigi/material-theme-issues/assets/8071683/aa79ed4f-67b3-4369-8cac-a02580a7f2c6)
